app/src/main/java/com/sopt/now/presentation/SignUpActivity.kt

```diff
@@ -53,15 +54,15 @@ class SignUpActivity : AppCompatActivity() {
         })
     }
 
-    private fun getSignUpRequestDto(): RequestSignUpDto {
+    private fun getSignUpData(): AuthData {
         val id = binding.etIdSignup.text.toString()
         val password = binding.etPwSignup.text.toString()
         val nickname = binding.etNicknameSignup.text.toString()
         val phoneNumber = binding.etMbtiSignup.text.toString()
-        return RequestSignUpDto(
-            authenticationId = id,
-            password = password,
-            nickname = nickname,
+        return AuthData(
+            id = id,
+            pw = password,
+            name = nickname,
             phone = phoneNumber
```

Comment:
```suggestion
        return AuthData(
            id = binding.etIdSignup.text.toString(),
            pw = binding.etPwSignup.text.toString(),
            name = binding.etNicknameSignup.text.toString(),
            phone = binding.etMbtiSignup.text.toString()
```

이렇게 불필요한 변수 생성없이 사용하면 코드 줄일 수 있을거 같아요